<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>881</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion services</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>851</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableView" name="tab_s">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>30</y>
         <width>451</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>ID_Service</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>heure d'ouverture</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lD">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>70</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="type_s">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="etat_S">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>130</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="heure_s_O">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTimeEdit" name="heure_S_F">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>210</y>
         <width>118</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>heure de fermeture</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Ajouter">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>290</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Modifier">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Supprimer">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>290</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="supp">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>320</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="tri">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string> tri</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>250</y>
         <width>71</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>ID Service</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>250</y>
         <width>51</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>250</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>250</y>
         <width>41</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>ALL</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Recherche">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="rech">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>320</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="stat">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Statistique</string>
       </property>
      </widget>
      <widget class="QPushButton" name="PDF">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>320</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>mail</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>691</width>
        <height>341</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QLineEdit" name="envoi">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>envoi Ã </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>object</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="objet">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>231</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>msg </string>
       </property>
      </widget>
      <widget class="QPushButton" name="sendMail">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>send mail</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>231</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
